Integrations/integration-MISP_V2.yml

```diff
@@ -710,6 +720,30 @@ script:
             'EntryContext': ec
         })
 
+
+    def add_sighting():
+        sighting = {
+            'sighting': 0,
+            'false-positive': 1,
+            'expiration': 2
+        }
+        kargs = {
+            'id': demisto.args().get('id'),
+            'uuid': demisto.args().get('uuid'),
+            'type': sighting.get(demisto.args().get('type'))
+        }
+        att_id = demisto.args().get('id', demisto.args().get('uuid'))
+        MISP.set_sightings(kargs)
+        demisto.results(f'Sighting \'{demisto.args().get("type")}\' added to attribute {att_id}')
+
+
+    def test():
+        if not MISP.test_connection():
+            demisto.error('MISP has not connected.')
```

Comment:
Why not use `return_error()` ? And please improve the error message to be more indicative

Reply:
it'll never get there anyway, PyMISP will throw an indicative error before.
If it does get there, it'll be odd and without any further information.

There's a debug tool that I'll try to add to this PR in MISP